tools/release/release.sh

```diff
@@ -117,7 +117,7 @@ version=${version:-${versionCandidate}}
 versionMajor=`echo ${version} | cut  -d "." -f1`
 versionMinor=`echo ${version} | cut  -d "." -f2`
 versionPatch=`echo ${version} | cut  -d "." -f3`
-nextPatchVersion=$((versionPatch + 2))
+nextPatchVersion=$((versionPatch + 1))
```

Comment:
Since we do not really do hotfix release currently I always have to change the value manually. This is a temporary change, maybe one day I will revert it.